app/controllers/api/articles_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::ArticlesController < ApplicationController
+  def index
+    articles = Article.all.sort_by(&:created_at).reverse
```

Comment:
This is a good example of logic that could be placed in the model and make use of scope. Not a blocker.

Reply:
fixed